make/RunTestsPrebuiltSpec.gmk

```diff
@@ -68,6 +68,9 @@ JAVA_FLAGS_SMALL := -XX:+UseSerialGC -Xms32M -Xmx512M -XX:TieredStopAtLevel=1
 BUILDJDK_JAVA_FLAGS_SMALL := -Xms32M -Xmx512M -XX:TieredStopAtLevel=1
 BUILD_JAVA_FLAGS := $(JAVA_FLAGS_BIG)
 
+# Express that testing needs additional dependencies
```

Comment:
You don't need to set this. In fact, it makes it impossible to set it from the command line.